src/Engines/Processors/shaderProcessor.ts

```diff
@@ -9,6 +9,7 @@ import { ShaderDefineExpression } from './Expressions/shaderDefineExpression';
 import { ShaderDefineArithmeticOperator } from './Expressions/Operators/shaderDefineArithmeticOperator';

```

Comment:
could you simply revert this file ??? and we ll merge it. It works beautifully well so far in WebGPU, so it should be the same in native , thanks a ton.

@syntheticmagus @CedricGuillemet @bghgary to keep on your side the integration snippet for native :-)

Reply:
Done